x-pack/plugins/observability_solution/investigate_app/public/components/add_note_ui/index.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import type { AuthenticatedUser } from '@kbn/core/public';
+import type { GlobalWidgetParameters, OnWidgetAdd } from '@kbn/investigate-plugin/public';
+import React from 'react';
+import { NoteWidgetControl } from '../note_widget_control';
```

Comment:
This is more like an organization question for the Note Widget that will follow up in another PR. Would we move the note widget control in the new add_note_ui folder?

Reply:
I think we can, up to you. I think we will rename a bunch of components as we work on this